File: AkitaModel/Portal/TestHistory.cs
using System.Collections.Generic;

namespace Skyware.Lis.AkitaModel.Portal;

/// <summary>
/// Represents test results over time for given Patient.
/// </summary>
public class TestHistory
{

    /// <summary>
    /// Patient.
    /// </summary>
    public Patient Patient { get; set; }

    /// <summary>
    /// The Test this summary is for.
    /// </summary>
    public Test Test { get; set; }

    /// <summary>
    /// The results of the test.
    /// </summary>
    public IList<TestResult> Results { get; set; }

}

File: AkitaMotelTS/src/Portal/TestHistory.ts
/**
 * Skyware.Lis.AkitaModel.Portal
 */

import { Patient } from '../Patient';
import { Test } from '../Test';
import { TestResult } from './TestResult';

/**
 * Represents test results over time for given Patient.
 */
export class TestHistory {
  /**
   * Patient.
   */
  patient?: Patient;

  /**
   * The Test this summary is for.
   */
  test?: Test;

  /**
   * The results of the test.
   */
  results?: TestResult[];
}
